From 2730d29934841b6d62f68f917af784b307eaa1a9 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Sun, 21 Jul 2024 16:14:23 -0700
Subject: [PATCH 11/11] HACK: thermal: Remove __counted_by from
 thermal_zone_device

This currently crashes clang when assertions are enabled.

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 include/linux/thermal.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/thermal.h b/include/linux/thermal.h
index c33f50177f51..ddcea0798578 100644
--- a/include/linux/thermal.h
+++ b/include/linux/thermal.h
@@ -203,7 +203,7 @@ struct thermal_zone_device {
 #ifdef CONFIG_THERMAL_DEBUGFS
 	struct thermal_debugfs *debugfs;
 #endif
-	struct thermal_trip trips[] __counted_by(num_trips);
+	struct thermal_trip trips[] /* __counted_by(num_trips) */;
 };
 
 /**
-- 
2.45.2

